File: level1-최대공약수와최소공배수.cpp
#include <string>
#include <vector>

using namespace std;

int gcd(int a, int b) {
    int mod = a%b;
    while(mod > 0) {
        a = b;
        b = mod;
        mod = a%b;
    }
    return b;
}

int lcm(int a, int b) {
    return (a*b)/gcd(a,b);
}

vector<int> solution(int n, int m) {
    vector<int> answer;
    int gcd_result = gcd(n, m);
    int lcm_result = lcm(n, m);
    answer.push_back(gcd_result);
    answer.push_back(lcm_result);
    return answer;
}

File: level1-최대공약수와최소공배수.py
def solution(n, m):
    answer = [gcd(n,m), lcm(n,m)]
 
    return answer

def gcd(a, b):
    mod = a%b
    while mod > 0:
        a = b
        b = mod
        mod = a%b
    return b
    
def lcm(a, b):
    return a*b//gcd(a,b)